hw/misc/aspeed_hace.c

```diff
@@ -104,6 +104,97 @@ static int do_hash_operation(AspeedHACEState *s, int algo)
     return 0;
 }
 
+static int do_hash_sg_operation(AspeedHACEState *s, int algo)
+{
+    hwaddr src, len, dest, buf_len;
```

Comment:
I think these should be uint32_t since they come from the register.

Reply:
Solved in latest force-push